recipes/nodejs/all/conanfile.py

```diff
@@ -1,48 +1,82 @@
 import os
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+import re
+from conans import __version__
+from conan import ConanFile
+from conan.tools.scm import Version
+from conan.tools.files import copy, get
+from conan.errors import ConanInvalidConfiguration
 
-required_conan_version = ">=1.33.0"
+from six import StringIO
+
+required_conan_version = ">=1.59.0"
 
 
 class NodejsConan(ConanFile):
     name = "nodejs"
     description = "nodejs binaries for use in recipes"
-    topics = ("conan", "node", "nodejs")
+    topics = ("node", "nodejs")
```

Comment:
repeating the name does not help the search :)

```suggestion
    topics = ("node", "javascript", "runtime")
```